builtin/logical/mysql/backend_test.go

```diff
@@ -220,14 +252,26 @@ func testAccStepConfig(t *testing.T, d map[string]interface{}, expectError bool)
 	}
 }
 
-func testAccStepRole(t *testing.T) logicaltest.TestStep {
+func testAccStepRole(t *testing.T, wildCard bool) logicaltest.TestStep {
+
+	pathData := make(map[string]interface{})
+	if wildCard == true {
+		pathData = map[string]interface{}{
+			"sql": testRoleWildCard,
+		}
+	} else {
+		pathData = map[string]interface{}{
```

Comment:
My thought was if you added and sql statement for creating users with wild card hosts. You would not provide a customer revoke sql statement. So the test for wild card should have no revoke sql to test the default revoke statement that uses wild card for a host. 

Unless you think I should add a customer revoke sql statement that is is the same as the default in the case the the user provided a revoke sql statement with wildcards as the host not knowing that the default case use the wildcard.